apps/emqx/test/emqx_shared_sub_SUITE.erl

```diff
@@ -365,6 +453,27 @@ ensure_config(Strategy, AckEnabled) ->
     emqx_config:put([broker, shared_dispatch_ack_enabled], AckEnabled),
     ok.
 
+ensure_group_config(Group2Strategy) ->
+    maps:foreach(
```

Comment:
maps:foreach is introduced in otp 24, but we need to make EMQX code compile on otp 23 too.